api/islandora_xacml_api.install

```diff
@@ -110,3 +110,83 @@ function islandora_xacml_api_update_7102(&$sandbox) {
   drupal_set_message(t('Please run the islandora_xacml_api_relationships_update_batch drush command.'));
   return t('Please run the islandora_xacml_api_relationships_update_batch drush command.');
 }
+
+/**
+ * Remove superfluous relationship from the RELS-INT.
+ *
+ * @see https://jira.duraspace.org/browse/ISLANDORA-1273
+ */
+function islandora_xacml_api_update_7103(&$sandbox) {
+  if (!variable_get('islandora_xacml_api_save_relationships', TRUE)) {
+    return t('Relationships are disabled, nothing to do.');
+  }
+  drupal_load('module', 'islandora');
+  $tuque = islandora_get_tuque_connection();
+  $rels_ext_uri = ISLANDORA_RELS_EXT_URI;
+
+  $query = <<<EOQ
+SELECT DISTINCT ?ds FROM <#ri> WHERE {
+  {
+    {
+      ?ds <{$rels_ext_uri}isManageableByRole> ?role
+    }
+    UNION {
+      ?ds <{$rels_ext_uri}isManageableByUser> ?user
+    }
+  }
+  ?obj <fedora-view:disseminates> ?ds
+}
+EOQ;
+
+  if (!isset($sandbox['total'])) {
+    $sandbox['progress'] = 0;
+    $sandbox['total'] = $tuque->repository->ri->countQuery($query, 'sparql');
+    if ($sandbox['total'] == 0) {
+      // Nothing to process.
+      $sandbox['#finished'] = 1;
+      return t('Nothing to fix.');
+    }
+    $sandbox['result_stash'] = array();
+  }
+
+  if (empty($sandbox['result_stash'])) {
+    $offset = 100;
+    $limited_query = $query . <<<EOQ
+LIMIT $offset
+EOQ;
+
+    $sandbox['result_stash'] = $tuque->repository->ri->sparqlQuery($limited_query);
+    if (empty($sandbox['result_stash'])) {
+      // Ran out of items early?
+      $sandbox['#finished'] = 1;
+      return t('Updated @count PID(s) and ran out of items early... Somebody manually updated a document?', array(
+        '@count' => $sandbox['total'],
+      ));
+    }
+  }
+
+  $result = array_pop($sandbox['result_stash']);
+  $datastream_uri = $result['ds']['value'];
+
+  $parts = explode('/', $datastream_uri);
+  $pid = $parts[0];
+  $ds = $parts[1];
+  $object_to_update = islandora_object_load($pid);
+
+  $manageable_by_user = 'isManageableByUser';
+  $manageable_by_role = 'isManageableByRole';
+  $object_to_update[$ds]->relationships->remove(ISLANDORA_RELS_EXT_URI, $manageable_by_user);
+  $object_to_update[$ds]->relationships->remove(ISLANDORA_RELS_EXT_URI, $manageable_by_role);
```

Comment:
Work is still moving on this https://github.com/Islandora/islandora_xacml_editor/pull/115